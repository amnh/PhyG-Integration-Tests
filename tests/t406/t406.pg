--input t88.tre, swap(tbr), inputting fasta files with missing and chopped data
--the taxa in these files were further reduced to 51 taxa and the root was set
set(graphssteepest:1)
set(seed:1634561640)
set(outgroup:"Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184")
read(fasta:"dec258_12Saln_missing.fas", "dec258_16Saln_chopped.fasta")
read(include:"Include_dec.txt")
read(newick:"../t88/t88.tre")
swap(tbr)
report("t406.csv", data, overwrite)
report("t406.tre", newick, graphs, overwrite)