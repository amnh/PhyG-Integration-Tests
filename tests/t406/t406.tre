(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:308.0,(((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:114.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:100.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:107.0)HTU60:54.0)HTU32:85.0,((((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:58.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:73.0)HTU33:35.0,((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:149.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:126.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:119.0)HTU59:125.0)HTU50:46.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:135.0,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:143.0)HTU54:39.0)HTU41:47.0)HTU49:41.0,((Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:143.0,Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:315.0)HTU35:66.0,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:71.0,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:81.0)HTU34:44.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:119.0,Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:102.0)HTU36:33.0)HTU42:84.0)HTU52:38.0)HTU37:33.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:149.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:197.0,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:134.0)HTU53:74.0)HTU43:39.0)HTU51:35.0)HTU44:46.0,((Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:165.0,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:33.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:29.0)HTU47:56.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:47.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:51.0,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:40.0)HTU56:37.0)HTU55:43.0)HTU48:20.0)HTU38:31.0,((Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:176.0,Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:223.0)HTU39:70.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:178.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:169.0,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:225.0)HTU40:56.0)HTU45:48.0)HTU57:35.0)HTU46:34.0)HTU58:156.0)HTU31[11276.0];
