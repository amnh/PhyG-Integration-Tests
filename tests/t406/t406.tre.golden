(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:559.5,(((((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:347.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:337.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:355.0)HTU54:98.0)HTU42:147.0,((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:395.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:358.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:371.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:457.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:473.5)HTU57:101.5)HTU33:123.5)HTU47:101.0)HTU39:102.5,(((Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:310.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:419.5,Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:439.0)HTU40:143.5)HTU38:110.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:315.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:468.0,Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:297.0)HTU52:188.0)HTU46:131.0)HTU41:112.5,(((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:349.0,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:2041.0,Eucarida_Dendrobranchiata_Penaeoidea_Penaeidae_Penaeus_californiensis_NC_012738:0.0)HTU51:1795.0)HTU34:132.0,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:349.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:477.5,Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:430.5)HTU49:156.0)HTU43:139.0)HTU44:101.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:274.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:262.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:243.5)HTU56:89.0)HTU55:119.5)HTU59:102.0)HTU58:106.0)HTU61:108.5)HTU36:118.5,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:347.5,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:388.5)HTU48:107.0)HTU35:98.5,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:358.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:361.0)HTU37:260.5)HTU45:109.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:395.5,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:346.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:338.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:367.5)HTU53:65.5)HTU60:99.5)HTU50:110.5)HTU62:460.5)HTU32[11339.0];
(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:559.5,(((((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:347.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:337.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:355.0)HTU54:98.0)HTU42:147.0,((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:395.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:358.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:371.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:457.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:473.5)HTU57:101.5)HTU33:123.5)HTU47:101.0)HTU39:102.5,(((Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:310.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:419.5,Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:439.0)HTU40:143.5)HTU38:110.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:315.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:468.0,Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:297.0)HTU52:188.0)HTU46:131.0)HTU41:112.5,(((Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:349.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:477.5,Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:430.5)HTU49:156.0)HTU43:139.0,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:307.0,(Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:2068.5,Eucarida_Dendrobranchiata_Penaeoidea_Penaeidae_Penaeus_californiensis_NC_012738:0.0)HTU34:1795.0)HTU51:132.0)HTU44:101.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:274.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:262.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:243.5)HTU56:89.0)HTU55:119.5)HTU59:102.0)HTU58:106.0)HTU61:108.5)HTU36:118.5,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:347.5,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:388.5)HTU48:107.0)HTU35:98.5,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:358.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:361.0)HTU37:260.5)HTU45:109.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:395.5,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:346.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:338.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:367.5)HTU53:65.5)HTU60:99.5)HTU50:110.5)HTU62:460.5)HTU32[11339.0];
